runbot_docker_orchest/models/runbot_build.py

```diff
@@ -0,0 +1,55 @@
+from odoo import models, fields, api
+import subprocess
+import uuid
+MAGIC_PID_RUN_NEXT_JOB = -2
+
+
+class RunbotBuild(models.Model):
+    _inherit = "runbot.build"
+    is_build_quayio = fields.Boolean("Build with image quay.io", readonly=True)
+
+    @api.depends('name', 'branch_id.name')
+    def _get_dest(self):
+        builds = self.filtered('branch_id.repo_id.is_quayio_image')
+        super(RunbotBuild, self - builds)._get_dest()
+        for build in builds:
+            build.dest = str(uuid.uuid4())
```

Comment:
Use the same logic from: 
 - https://github.com/odoo/runbot/blob/4a558741e01b206152ed10f5062d0fc94121fed5/runbot/models/build.py#L212

build.id
nickname = docker_image.tag
buld.name[:6] <- docker_image.sha